### PR TITLE
Blog post Prague 2019 talks announced, CFV, tickets, shirts

### PR DESCRIPTION
**Pull request by thatdocslady**

### CODE REVIEW

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+--------------------
+
+Every year, we look to bring a wide range of voices to the Write the Docs stage. Because the role of "documentarian" looks so different to each of us, we aim for a line-up that spans a good mix of practical, philosophical, and technical topics – with the odd whimsical one thrown in for good measure. We're really excited about the presentations we've got this year – we hope you are too!
+
+.. datatemplate::
+   :source: /_data/{{year}}.{{shortcode}}.speakers.yaml
+   :template: {{year}}/speakers-simple-list.rst
+   :include_context:
+
+Head over to the `Speakers page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/speakers/>`_ to see the full abstracts. We hope there’s something there for everyone to enjoy, and also something that will broaden your horizons a bit.
```
Comment (plaindocs) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst]: I'd go better than "we hope" :-D

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+Conference shirts
+-----------------
+
+Following the success of our pilot last year, once again we'll be selling t-shirts separately from your conference ticket. This is due partially to the logistics of sizing, ordering, and managing stock, but also to help reduce waste from the inevitable leftovers that we find ourselves with every year.
+
+Head over to our Write the Docs Prague 2019 `Pop-Up Shop <https://teespring.com/wtd-prague-2019>`__ and order this year's shirt. We have several cuts and fabrics available, and we believe that you'll be able to find the size and style that fits you. The campaign will run until **30th September** in case anyone wants to order their shirts after the conference, but if you want to sport your new threads at the conference, we recommend to order at least 2-3 weeks in advance.
+
+And don't worry, you'll still have our delightful Write the Docs Prague 2019 bags as a souvenir from the conference even if you decide not to order a shirt.
+
+Thanks to our sponsors
```
Comment (plaindocs) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst]: Twice is maybe too much?

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+----------------------
+
+Thanks to our sponsors
+----------------------
+
+We are so grateful to have our sponsors help in bringing these events to life every year. Thanks sincerely to the following companies for supporting the Write the Docs community:
```
Comment (plaindocs) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst]:
```suggestion
We are so grateful to have our sponsors help in bringing these events to life every year. Thanks to the following companies for supporting the Write the Docs community:
```

Just cos when you say sincerely I assume you're not :-p

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+`Umbraco <https://umbraco.com/>`_
+
+`Buildkite <https://buildkite.com/>`_
+
+Want to support our community? Head over to our `sponsorship prospectus <https://www.writethedocs.org/conf/prague/2019/sponsors/prospectus/>`_ for details on how your company can get involved! We have several types of sponsorship available, including tables at the job fair that we'll be hosting on Tuesday.
+
+Stay tuned
+----------
+
+Over the next weeks we'll continue to publish updates about the various events and activities that make Write the Docs events such a unique experience, as well as the presentation schedule, trip planning tips.
```
Comment (plaindocs) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst]:
```suggestion
Over the next weeks we'll continue to publish updates about the various events and activities that make Write the Docs events such a unique experience, as well as the presentation schedule, trip planning tips, etc.
```

**Review by plaindocs — COMMENTED** [review]
@thatdocslady it would make life a lot easier if you picked one of the following:

- added the blog post directly in the speaker PR 
- branched from the branch used by the speaker PR and made the PR against that

This way the speaker data is broken till we merge the other PR.

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst (hunk 1 above)
Comment (thatdocslady) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst]:
```suggestion
Head over to the `Speakers page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/speakers/>`_ to see the full abstracts. We believe there’s something there for everyone to enjoy, and also something that will broaden your horizons a bit.
```

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst (hunk 2 above)
Comment (thatdocslady) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst]:
```suggestion
```

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+-----------------
+
+Following the success of our pilot last year, once again we'll be selling t-shirts separately from your conference ticket. This is due partially to the logistics of sizing, ordering, and managing stock, but also to help reduce waste from the inevitable leftovers that we find ourselves with every year.
+
+Head over to our Write the Docs Prague 2019 `Pop-Up Shop <https://teespring.com/wtd-prague-2019>`__ and order this year's shirt. We have several cuts and fabrics available, and we believe that you'll be able to find the size and style that fits you. The campaign will run until **30th September** in case anyone wants to order their shirts after the conference, but if you want to sport your new threads at the conference, we recommend to order at least 2-3 weeks in advance.
+
+And don't worry, you'll still have our delightful Write the Docs Prague 2019 bags as a souvenir from the conference even if you decide not to order a shirt.
+
+Thanks to our sponsors
+----------------------
```
Comment (thatdocslady) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst]:
```suggestion
```

**Review by thatdocslady — COMMENTED** [review]
Implemented feedback

**Comment by thatdocslady** [comment]
@plaindocs would you mind pulling in my changes to the main speakers PR? My git-foo stands humbled...

**Comment by plaindocs** [comment]
I looked at changing the PR branch to the speaker PR branch instead of master, but there is a whole host of other changes there as well. I could just cherry-pick these three commits I guess, but probably the easiest is just to merge that speaker PR as soon as we update from Grorg for the last time, and merge this after.

**Comment by thatdocslady** [comment]
No worries, that's what I originally proposed ;)

I'l wait until you merge your PR, rebase from master and test the output, and then if no other feedback then I'll merge mine. Assuming you'll do this tomorrow, I should be online and responsive to GH notifications.

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+Diversity scholarship tickets are **free**, and we hope to welcome more people from all walks of life to join our community. The deadline for applications is **7th July**.
+
+Please share this campaign with your social networks and encourage folks to apply if they are interested to join us!
+
+Conference shirts
+-----------------
+
+Following the success of our pilot last year, once again we'll be selling t-shirts separately from your conference ticket. This is due partially to the logistics of sizing, ordering, and managing stock, but also to help reduce waste from the inevitable leftovers that we find ourselves with every year.
+
+Head over to our Write the Docs Prague 2019 `Pop-Up Shop <https://teespring.com/wtd-prague-2019>`__ and order this year's shirt. We have several cuts and fabrics available, and we believe that you'll be able to find the size and style that fits you. The campaign will run until **30th September** in case anyone wants to order their shirts after the conference, but if you want to sport your new threads at the conference, we recommend to order at least 2-3 weeks in advance.
```
Comment (mxsasha) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst:75]: Is "sport your new threads" a sufficiently well-known expression?

**Review by mxsasha — APPROVED** [review]

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst (hunk 6 above)
Comment (thatdocslady) [reply docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst:75]: I don't think there's a problem with it, do you have strong feelings about it?

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst [hunk 7]
```diff
@@ -0,0 +1,100 @@
+:template: {{year}}/generic.html
+
+.. post:: June 24, 2019
+   :tags: prague-2019, speakers, tickets, shirts, sponsors
+
+Announcing speakers and talks, call for volunteers, ticket update, shirts on sale
```
Comment (thatdocslady) [inline docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst]:
```suggestion
Announcing Speakers and Talks, Call for Volunteers, Ticket Update, Shirts on Sale
```

File: docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst (hunk 6 above)
Comment (mxsasha) [reply docs/conf/prague/2019/news/talks-volunteers-tickets-shirts.rst:75]: No strong feelings. If you think it's fine, let's leave it in.

**Pull request merged**